test/conformance/envpropagation_test.go

```diff
@@ -26,15 +26,17 @@ import (
 	"github.com/knative/serving/test"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	. "github.com/knative/serving/pkg/reconciler/testing"
 )
 
 const (
 	testKey   = "testKey"
 	testValue = "testValue"
 )
 
-// TestSecretsFromEnv verifies propagation of Secrets through environment variables.
-func TestSecretsFromEnv(t *testing.T) {
+// TestSecretsViaEnv verifies propagation of Secrets through environment variables.
+func TestSecretsViaEnv(t *testing.T) {
```

Comment:
As mentioned in the issue we should move this test to our e2e test suite instead of our conformance test suite. The use of the Kube client in creating the secret and cleaning up does not really fit our requirements for conformance tests.

Reply:
Not new, follow up?

Reply:
Created an issue to track. We can address this in a follow-up https://github.com/knative/serving/issues/3984